CHANGES.md

```diff
@@ -1,5 +1,59 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.7.0-Beta
+
+### Core API significant improvements
+
+* New `Channel` implementation with significant performance improvements across the API (#3621).
+* New `select` operator implementation: faster, more lightweight, and more robust (#3020).
+* `Mutex` and `Semaphore` now share the same underlying data structure (#3020).
+* `Dispatchers.IO` is added to K/N (#3205)
+  * `newFixedThreadPool` and `Dispatchers.Default` implementations on K/N were wholly rewritten to support graceful growth under load (#3595).
+* `kotlinx-coroutines-test` rework:
+  - Add the `timeout` parameter to `runTest` for the whole-test timeout, 10 seconds by default (#3270). This replaces the configuration of quiescence timeouts, which is now deprecated (#3603).
+  - The `withTimeout` exception messages indicate if the timeout used the virtual time (#3588).
+  - `TestCoroutineScheduler`, `runTest` and `TestScope` API are promoted to stable (#3622).
```

Comment:
```suggestion
  - `TestCoroutineScheduler`, `runTest`, and `TestScope` API are promoted to stable (#3622).
  - `runTest` now also fails if there were uncaught exceptions in coroutines not inherited from the test coroutine (#1205).
```

Reply:
I don't see the suggested new line anywhere.

Reply:
Oh, I see -- I've applied the comma, but missed the next line